src/js/context/JSONSchemaFormDataProvider.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable class-methods-use-this */
+import type { UiSchema } from "react-jsonschema-form";
+
+import ContextProvider, { type ContextProviderConfig } from "./ContextProvider";
+
+interface JSONSchemaFormDataProviderConfig extends ContextProviderConfig {
```

Comment:
JSONSchemaForm is a concept that includes both data and UI - we may want to move it to `WoVe`. Something to discuss.

Reply:
Otherwise, AVe, could use it too